community/community-engines/flink-batch/src/main/scala/org/apache/mahout/flinkbindings/drm/CheckpointedFlinkDrm.scala

```diff
@@ -62,17 +62,16 @@ class CheckpointedFlinkDrm[K: ClassTag:TypeInformation](val ds: DrmDataSet[K],
 
   // this is extra I/O for each cache call.  this needs to be moved somewhere where it is called
   // only once.  Possibly FlinkDistributedEngine.
-  GlobalConfiguration.loadConfiguration(mahoutHome + "/conf/flink-config.yaml")
+  GlobalConfiguration.loadConfiguration(mahoutHome + "/conf/")
```

Comment:
getting a failure here: with `mvn clean package install`.  We've moved The `Flink` Module to community so that any members of the community can make use of it, see what we ran into, etc.  Our main issue was with flink's greedy execution of code re-written by the optpimizer, and Overflowing Memory mid-expression which should not have been necessary:

E.g.  X:= AX.tX * BB.t - c
when put through the optimizer will come out as 
        A:= SelfSq(X)AB.tB -c

Something similar was happening, i cant remember exactly, but the second `B.tB` woule be expanded into memory.. I may be way off, It was the same for iterative algorithms though, the DAG would compute at each iteration for certain expressions, which were meant to be checkpoint and cached.  These Checpoints would Eagerly evaluate and crash Iterative Algos like DSSVD, and DSPCA.


I', unsure of the current state of Flink Batch, but 

is our simple explanatio

```org.apache.mahout.flinkbindings.DrmLikeOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalCon```

Reply:
verified missing `./conf/flink-config.yaml` :
```
[INFO] Forking ScalaTest via: /bin/sh -c cd '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch' && 'java' '-Djava.home=/Library/Java/JavaVirtualMachines/jdk1.8.0_201.jdk/Contents/Home/jre' '-Djdk.lang.Process.launchMechanism=FORK' '-Dmahout.home=/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/test-classes' '-Dbasedir=/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch' '-Xmx4g' 'org.scalatest.tools.Runner' '-R' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/classes /Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/test-classes' '-o' '-fWDF' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/surefire-reports/TestSuite.txt' '-u' '/Users/colleenpalumbo/sandbox/mahout/community/community-engines/flink-batch/target/surefire-reports/.'
Discovery starting.
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Discovery completed in 438 milliseconds.
Run starting. Expected test count is: 114
DrmLikeOpsSuite:
org.apache.mahout.flinkbindings.DrmLikeOpsSuite *** ABORTED ***
  org.apache.flink.configuration.IllegalConfigurationException: The Flink config file '/Users/colleenpalumbo/sandbox/mahout/conf/flink-conf.yaml' (/Users/colleenpalumbo/sandbox/mahout/conf) does not exist.
  at org.apache.flink.configuration.GlobalConfiguration.loadConfiguration(GlobalConfiguration.java:124)
  at org.apache.flink.configuration.```